crashsimilarity/cache.py

```diff
@@ -0,0 +1,89 @@
+import json
+import pickle
+from abc import abstractmethod
+
+import logging
+
+from crashsimilarity import utils
+
+
+class BaseCache(object):
+    def __init__(self, name, file_name=None):
```

Comment:
No, it isn't. The idea is to have possibility to specify file name where cache will be dumped. It can be default and can be not.